lib/bump/bump.ts

```diff
@@ -218,7 +247,7 @@ export class AutoBump extends cdk.Construct {
               // checked the code. Instead we define a variable that we will switch all other lines on.
               // tslint:disable-next-line:max-line-length
               `git describe --exact-match HEAD && { echo "No new commits."; export SKIP=true; } || { echo "Changes to release."; export SKIP=false; }`,
-              `$SKIP || { ${bumpCommand}; }`,
+              `$SKIP || { ${bumpCommand !== 'disable' ? bumpCommand : 'echo Bump is disabled'}; }`,
```

Comment:
why not just `undefined` to indicate no bump command?

Reply:
I'd love to, but there is a default: `/bin/bash ./bump.sh`.

We had the same problem with `scheduleExpression`:

https://github.com/awslabs/aws-delivlib/blob/a0168d7cdd15dbb5d51bc1a8236048c873906fc9/lib/bump/bump.ts#L243-L253

Part of the reason I want to refactor...